### PR TITLE
Nathrezim Ethnicity

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Created a custom ethnicity for the Nathrezim.
  - The primary inspiration and reference for the ethnicity are the [new Dreadlord/Dreadqueen models added in Chains of Domination](https://ptr.wowhead.com/news/new-dreadlord-models-in-chains-of-domination-dreadlords-return-in-patch-9-1-321893).
- Included 7 facial markings and a custom lipstick decal.
- Claws, horns, and wings added by @zumbak04.
- There's a 1% chance for Nathrezim to gain the "[Detheroc Gene](https://i.ytimg.com/vi/SdgdLd0dBug/maxresdefault.jpg)". This is intended.

## Tasks:
- [x] Localize Nathrezim markings.
- [x] Create a custom ethnicity for the Nathrezim.
- [x] Create "sub-ethnicities" for the Nathrezim.
- [x] Make Nathrezim bald and clean-shaven.
- [x] Disable eyebrows for male Nathrezim but leave them enabled for females.
- [x] Add claws for Nathrezim.
- [x] Make Nathrezim look older (around 30-40).
  - [x] Add `very_fast_aging` gene for `old_1.`
  - [x] Add `very_fast_aging` gene for `old_2.`
  - [x] Add `very_fast_aging` gene for `old_3.`
  - [x] Add `very_fast_aging` gene for `old_4.`
- [x] Add custom lipstick decal for Nathrezim.
- [x] Add `avg_spacing_low_thickness` eyebrows to female Nathrezim.
- [x] Add facial markings for male Nathrezim.
  - [x] `decal_nathrezim_male_01`
  - [x] `decal_nathrezim_male_02`
  - [x] `decal_nathrezim_male_03`
  - [x] `decal_nathrezim_male_04`
- [x] Add facial markings for female Nathrezim.
  - [x] `decal_nathrezim_female_01`
  - [x] `decal_nathrezim_female_02`
  - [x] `decal_nathrezim_female_03`

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
### There are a few ways to test the ethnicity:

**A**. Load up the 605 bookmark and look for Nathrezim characters using the Character Finder.
<details><summary>Click to expand</summary>

image

</details>

**B.** Open up the Ruler Designer and generate characters with the Nathrezim ethnicity.

<details><summary>Click to expand</summary>

image

</details>

**C.** With [debug mode enabled](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode), open up the Portrait Editor and generate characters with the `nathrezim_ethnicity`. This way you can also view the individual tints used by their culture.

<details><summary>Click to expand</summary>

image

</details>

### Providing feedback:

Provide feedback on the genes and markings, i.e. the overall appearance of the ethnicity. You may consider the following points when writing your review:
- Do they look like the dreadlords in WoW?
- Are the generations unique? _(This is not as important since they are demons.)_
- In your subjective opinion, are you happy with how they look?

### CODE REVIEW

**Review by Remerod — CHANGES_REQUESTED** [review]
So I've played around a bit. (using debug_mode, forgot to turn it off).

<details>
<summary>Click to expand</summary>

I do like the look of the Nathrezim a lot, they look very iconic

image
image
One sad issue though is that one 60-80% of the portraits the horns are hard to see, if not almost impossible. Although this issue might be not fixable for now and will perhaps be resolved in Royal Courts when the characters are in the court hall, because it would be sad if we make the horns and we don't even get to see them.

image
image
image
I like that there is a great variation around and that some have complexity(Wrinkles) to their face. It reminds me of the look that they have in wc3 (like in the one i posted above) witch i really like. Although the complexity(wrinkles) almost never occur in the randomize appearance in the character designs.

image
But at least we get us some chunky Nathrezim in the randomize appearance option :D

One of the stuff i would like to be added as face options for Nathrezim are on the issue here https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/646 that i wrote. I believe that have some Nathrezim with these option would add to their diversity and demonic look.

I overall really like this. The Horns, wings, ears, facial structures, eyes and markings make them very believable as Nathrazim and i would say that they are great and workable for the mod as they are.. **but**

image
_The legs_

Nathrazim have always had Iconic goat looking legs.
image
image
image
(also this one has some hair on the arm witch look cool, it could be a feature for a select few Nathrazim i think)

So i would argue that we should fix their legs, I do understand that the legs are not shown a great deal, for now. But with the Royal Court incoming i think that will greatly change. And their beautiful hairy goat legs are very iconic for the Nathrazim, and i think they should probably have them. 
</details>

File: common/culture/cultures/demonic.txt [hunk 1]
```diff
@@ -336,6 +336,8 @@
 	nathrezim = {
```
Comment (zumbak04) [inline common/culture/cultures/demonic.txt:336]: New markings have to be localized.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/culture/cultures/demonic.txt (hunk 1 above)
Comment (ercarp) [reply common/culture/cultures/demonic.txt:336]: Done in 7c279b049bdbbb55a659c447dd9475a5bbe48b63.

**Comment by ercarp** [comment]
> One sad issue though is that one 60-80% of the portraits the horns are hard to see, if not almost impossible. Although this issue might be not fixable for now and will perhaps be resolved in Royal Courts when the characters are in the court hall, because it would be sad if we make the horns and we don't even get to see them.

This is something I also voiced concerns about, but @zumbak04 and I decided that it was more important for them to be very tall, as the Nathrezim are known to be. We also attempted to fix it by changing the camera view in the character panel, but @zumbak04 said it made everyone else look too small.

You can still see their horns in the smaller character icons, spouse view, events, and when playing as a Nathrezim yourself.

<details><summary>Click to expand</summary>

image
image
image

</details>

> Nathrazim have always had Iconic goat looking legs.
> So i would argue that we should fix their legs, I do understand that the legs are not shown a great deal, for now.

They're currently using the same goat legs as our Draenei ethnicity, which I think is good enough—at least for the time being—for something that we cannot see 99% of the time.

<details><summary>Click to expand</summary>

image

</details>

> (also this one has some hair on the arm witch look cool, it could be a feature for a select few Nathrazim i think)

I think this is something we could come back to later when we create an ethnicity for Satyrs, as they also have furry legs and arms.

<details><summary>Click to expand</summary>

image

</details>

**Review by Remerod — APPROVED** [review]
I hear your Arguments @ercarp ! I still would hope that we'd could get something like I this that i proposed https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/646 But for now i agree with your arguments, and I see no reason to not approve ;)

**Comment by ercarp** [comment]
> I hear your Arguments @ercarp ! I still would hope that we'd could get something like I this that i proposed #646 But for now i agree with your arguments, and I see no reason to not approve ;)

I replied to you over there and I do agree with could do with some spikes but it's not really my area. I don't know how to import things like that into GoA. Also since similar spikes are utilized by at least three ethnicities, this would probably be better handled in a different PR rather than the Nathrezim specifically.

File: common/genes/wc_gene_facial_markings.txt [hunk 2]
```diff
@@ -845,6 +845,225 @@
...
+						priority = 10
+					}
+				}
+				female = male
+				boy = male
+				girl = male
+			}
+		}
+
+		gene_facial_markings_2 = {
```
Comment (zumbak04) [inline common/genes/wc_gene_facial_markings.txt:1033]: The gene need localization too.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/genes/wc_gene_facial_markings.txt (hunk 2 above)
Comment (ercarp) [reply common/genes/wc_gene_facial_markings.txt:1033]: Added in 1fc763f0705c037c6b51d4784753c5e10e3d8d9e.

<details><summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — APPROVED** [review]
LGTM! Great job!

**Review by Rhysheart — COMMENTED** [review]
They do look alot like Dreadlords. They have alot of little variants when randomizing with the Ruler Designer. I would be pretty happy with how they look, I think the effort is clear and a real good job. - Some clipping is forgivable in my opinion, as they do look really good.

unknown (6)

**Review by Darknight367 — APPROVED** [review]
The models' looking good, though like many other models their sizes makes most of their features not visible in the small window of the portraits. Great job overall!

**Review by Rhysheart — APPROVED** [review]

**Pull request merged**